### PR TITLE
removing instruction number 12

### PR DESCRIPTION
**Pull request by davidTocineta**
## Pull Request Title

Brief description about what this PR does and [which ISSUE-### is closed](https://link-to-your-ticket)

## Screenshots

|         | before | after |
| ------- | ------ | ----- |
| desktop |        |       |
| mobile  |        |       |

**Steps to reproduce:**

**Expected behavior**

### CODE REVIEW

**Review by eamzea — CHANGES_REQUESTED** [review]
# Changes needed / Cambios requeridos

- Improve PR description / Mejorar la descripción de la PR
- Add labels to PR / Agregar etiquetas al PR
- Add Project to PR / Agregar projecto al PR
- Link issue with PR / Enlazar tarea al PR

# Comments for future tasks / Comentarios para tareas futuras
- Commit "fix(remove)removing instruction number 12" -> should be "fix(instructions): removing instruction number 12"

File: images/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
     > example: `skyline added` -> `[imageName] -> added`.
 
-12. Agregar a Edgar o Andrés como Reviewer
+12.
```
Comment (abrilmunozc) [inline images/README.md:44]: Identation

Comment (abrilmunozc) [inline images/README.md:44]: you only remove de instruction but not the number

**Review by abrilmunozc — CHANGES_REQUESTED** [review]
You only removed the instruction but not the number 12

**Pull request closed without merging**